Automattic/jetpack
# Dashboard: avoid React warnings; use className instead of class

#### Changes proposed in this Pull Request:

* Tiny changes, just to avoid React warnings.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Not much to test here; try to go to Jetpack > Dashboard and Jetpack > Settings, and ensure you do not see a warning about `class` not being valid.
